contributing.md

```diff
@@ -13,10 +13,14 @@ machine to develop Bazel and, when you've made a patch, how to submit it.</p>
 In general, we prefer contributions that fix bugs or add features (as opposed to
 stylistic, refactoring, or "cleanup" changes).
 
-Please check with us on the [dev
-list](https://groups.google.com/forum/#!forum/bazel-dev) before investing a lot
-of time in a patch. Meet Bazel maintainers and other contributors on
-[Slack](https://slack.bazel.build).
+Please check with us on the [dev list](
+https://groups.google.com/forum/#!forum/bazel-dev) before investing
+a lot of time in a patch. You might also meet Bazel maintainers and
+other contributors on [Slack](https://slack.bazel.build). In general,
+if there is a Bazel team member assigned to an issue, someone is working
```

Comment:
Quite often, "assigned" is different from "actively working on it". For example, I might assign a bug to someone else because they should know about the bug and they are familiar with the area.
If someone else wants to work on the issue, they can just add a comment to it.

Reply:
I rewrote the entire section. PTAL